### PR TITLE
docs: Fix a few typos / wording issues

### PR DESCRIPTION
**Pull request by jamilbk**
Just a few things I found re-reading through over the weekend.

### CODE REVIEW

File: website/src/app/kb/use-cases/readme.mdx [hunk 1]
```diff
@@ -21,11 +21,11 @@ a Site.
 
 - [Block malicious DNS](/kb/use-cases/secure-dns): Use Firezone to improve your
   team's Internet security by blocking DNS queries to known malicious domains.
-- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access to a
-  Google Cloud VPC using Terraform to deploy multiple Gateways.
-- [Route through a public IP](/kb/use-cases/nat-gateway): Route some of your team's
-  traffic through a single, static IP address to use services that require
-  source traffic to come from specified IP allowlists.
+- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access into
+  your VPC using multiple deploys with optional Terraform example.
```
Comment (bmanifold) [inline website/src/app/kb/use-cases/readme.mdx]: Was the word `deploys` meant to be `gateways`?

Comment (jamilbk) [inline website/src/app/kb/use-cases/readme.mdx]:
```suggestion
  your VPC using multiple deploys with optional Terraform example.
```
```suggestion
  your VPC using multiple gateways with optional Terraform example.
```

Comment (jamilbk) [reply website/src/app/kb/use-cases/readme.mdx]: 👍

File: website/src/app/kb/use-cases/readme.mdx [hunk 2]
```diff
@@ -21,11 +21,11 @@ a Site.
 
 - [Block malicious DNS](/kb/use-cases/secure-dns): Use Firezone to improve your
   team's Internet security by blocking DNS queries to known malicious domains.
-- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access to a
-  Google Cloud VPC using Terraform to deploy multiple Gateways.
-- [Route through a public IP](/kb/use-cases/nat-gateway): Route some of your team's
-  traffic through a single, static IP address to use services that require
-  source traffic to come from specified IP allowlists.
+- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access into
+  your VPC using multiple gateways with optional Terraform example.
```
Comment (bmanifold) [inline website/src/app/kb/use-cases/readme.mdx]:
```suggestion
  your VPC using multiple Gateways with optional Terraform example.
```

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**